docs/Reference/EthSigner-CLI.md

```diff
@@ -3,7 +3,17 @@ description: EthSigner command line interface reference
 
 # EthSigner Command Line
 
-This reference describes the syntax of the EthSigner Command Line Interface (CLI) options.
+This reference describes the syntax of the EthSigner Command Line Interface (CLI) options. EthSigner 
```

Comment:
I'd suggest keeping this introduction as vague as possible, to mitigate the need to updated it when the Azure value PR lands.

Suggestion: `This reference describes the syntax of the EthSigner Command Line Interface (CLI) options. EthSigner signs transactions with a key provided from any of these sources:`

Reply:
Fixed